src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public String getSalaryInfo(String[] names, String[] data,
+                                              String dateFrom, String dateTo)
+            throws DateTimeParseException {
+        int wagePerHour;
+        int workingHours;
+        LocalDate dateStart = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateEnd = LocalDate.parse(dateTo, formatter);
+        LocalDate workDayDate;
```

Comment:
There is no reason to declare these variables in advance.
```suggestion
        LocalDate dateStart = LocalDate.parse(dateFrom, formatter);
        LocalDate dateEnd = LocalDate.parse(dateTo, formatter);
```

Reply:
Removed